modules/nextflow/src/main/groovy/nextflow/config/ConfigParserFactory.groovy

```diff
@@ -33,7 +33,7 @@ import nextflow.config.parser.v2.ConfigParserV2
 class ConfigParserFactory {
 
     static ConfigParser create() {
-        final parser = SysEnv.get('NXF_SYNTAX_PARSER', 'v1')
+        final parser = Const.getSyntaxParser()
```

Comment:
The use of `Const` for helper methods is mistake I've made at the beginning. Think it would better to no propagate it. `NF` could be a better place for this